bot/cogs/documentation.py

```diff
@@ -5,6 +5,7 @@
 import aiohttp
 import bleach
 import stackexchange
+from stackexchange import Site
```

Comment:
This isn't necessary, it only clutters imports, using `stackexchange.Site` is fine